tests/cover/test_conjecture_minimizer.py

```diff
@@ -17,15 +17,18 @@
 
 from __future__ import division, print_function, absolute_import
 
+from random import Random
```

Comment:
The minimizer is nominally randomized, even if the current implementation happens to be deterministic.